src/java/fr/paris/lutece/portal/web/stylesheet/StyleSheetFileServlet.java

```diff
@@ -90,10 +85,14 @@ protected void processRequest( HttpServletRequest request, HttpServletResponse r
                 response.setContentType( ( strMimetype != null ) ? strMimetype : "application/octet-stream" );
                 response.setHeader( "Content-Disposition", "attachement; filename=\"" + stylesheet.getFile( ) + "\"" );
 
-                OutputStream out = response.getOutputStream( );
-                out.write( stylesheet.getSource( ) );
-                out.flush( );
-                out.close( );
+                try ( OutputStream out = response.getOutputStream( ) )
+                {
+                    out.write( stylesheet.getSource( ) );
+                }
+                catch ( IOException e )

```

Comment:
Wouldn't it be better to respond to the client with a proper HTTP 500 instead of failing silently ?

Reply:
Will be corrected